source/module_esolver/esolver_ks.cpp

```diff
@@ -84,6 +70,23 @@ namespace ModuleESolver
     void ESolver_KS<T, Device>::Init(Input& inp, UnitCell& ucell)
     {
         ESolver_FP::Init(inp,ucell);
+
+        //------------------Charge Mixing------------------
+        p_chgmix->set_mixing(INPUT.mixing_mode,
+                             INPUT.mixing_beta,
+                             INPUT.mixing_ndim,
+                             INPUT.mixing_gg0,
+                             INPUT.mixing_tau);
```

Comment:
I am not sure here about whether it is appropriate to directly pass variable from Input class to inner class the Charge_Mixing, will GlobalV be a better choice? I mean from Input to GlobalV, then pass variables between GlobalV and Charge_Mixing. @dyzheng

Reply:
Values of class INPUT should not used in other codes, only input_conv.cpp can access these variables.